# camera_config.yaml
name: "cam3"
topic: "/cam3/color/image_raw"
H: [[1, 0, 0, 0], [0, 1, 0, 0], [0, 0, 1, 0], [0, 0, 0, 1]]
camera_matrix:
  [
    [1.369234645523975e+03, 0, 9.751224513237420e+02],
    [0, 1.369034167040666e+03, 5.491281644489998e+02],
    [0, 0, 1],
  ]
distortion_coefficients:
  [
    0.026326173154487,
    0.601765501776712,
    0.004923190481130,
    -0.001083377739232,
    -2.198008325063289,
  ]
serial_no: "819612071033"
